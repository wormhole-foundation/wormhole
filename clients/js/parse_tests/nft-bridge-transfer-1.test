010000000000000000010000000100010000000000000000000000000000000000000000000000000000000000000004000000000277bb0b0001000000000000000000000000000000000000000000000000000000000000000400010000000000000000000000000000000000000000000000000000000000464f4f0000000000000000000000000000000000000000000000000000000000424152000000000000000000000000000000000000000000000000000000000000000a0a676f6f676c652e636f6d0000000000000000000000000000000000000000000000000000000000000004000a