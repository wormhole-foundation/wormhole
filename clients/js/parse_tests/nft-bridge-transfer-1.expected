{
  version: 1,
  guardianSetIndex: 0,
  signatures: [],
  timestamp: 1,
  nonce: 1,
  emitterChain: 1,
  emitterAddress: '0x0000000000000000000000000000000000000000000000000000000000000004',
  sequence: 41401099n,
  consistencyLevel: 0,
  payload: {
    module: 'NFTBridge',
    type: 'Transfer',
    tokenAddress: '0x0000000000000000000000000000000000000000000000000000000000000004',
    tokenChain: 1,
    tokenSymbol: 'FOO',
    tokenName: 'BAR',
    tokenId: 10n,
    tokenURI: 'google.com',
    toAddress: '0x0000000000000000000000000000000000000000000000000000000000000004',
    chain: 10
  },
  digest: '0xe2a7a0f5d67018c74683851fca870403455f89d9f474e2af104740e31a00da63'
}
